routes/drones.js

```diff
@@ -1,37 +1,41 @@
 const express = require('express');
 
-// require the Drone model here
+const Drone = require('../models/Drone.model')
 
 const router = express.Router();
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+router.get('/drones/', (req, res, next) => {
+  Drone.find().then((allDronesFromDb) => {
+    res.render('drones/list', {myDrones :allDronesFromDb});
+  })
+  .catch(error => console.log("Error while getting the drones from the DB: ", error));
 });
 
 router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+  res.render('drones/create-form')
 });
 
 router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
```

Comment:
What you did is correct, the only thing you forgot was to add a `.catch()` after `.then()` in case an error occurs.

I would also suggest loading the values  from `req.body` inside variables. In general, I think it's better practice
```
const {name, propellers, maxSpeed} = req.body;

Drone.create({name, propellers, maxSpeed})
.then(()=>{res.redirect('/drones')
.catch(error => console.log("Error while adding the drone in the DB: ", error));